extensions/sql-database-projects/src/dialogs/deployDatabaseDialog.ts

```diff
@@ -86,6 +90,21 @@ export class DeployDatabaseDialog {
 				this.tryEnableGenerateScriptAndOkButtons();
 			});
 
+			this.loadProfileButton = this.createLoadProfileButton(view);
+			this.sqlCmdVariablesTable = view.modelBuilder.table().withProperties({
```

Comment:
What does the empty state look like (no profile loaded and no sqlcmdvars in the sqlproj)?  Do you think it should be an empty table or no table at all?

Reply:
This is how the empty state looks  (no profile loaded and no sqlcmdvars in the sqlproj):
<img width="414" alt="Screen Shot 2020-06-17 at 9 57 54 AM" src="https://user-images.githubusercontent.com/31145923/84926996-0cc92500-b081-11ea-8920-eb8c5ac2bbe1.png">

SSDT doesn't show a table in this case:
<img width="643" alt="Screen Shot 2020-06-17 at 9 58 51 AM" src="https://user-images.githubusercontent.com/31145923/84927070-2b2f2080-b081-11ea-9134-424bd76ceaef.png">

Reply:
Looks good!  I was hoping for no table.